test/e2e/kcm_operator.go

```
@@ -0,0 +1,233 @@
+		_, err := testlib.DebugNodeWithChroot(oc, leaderKcm, "mv", "/home/kube-apiserver-pod.yaml", "/etc/kubernetes/manifests/")
+		o.Expect(err).NotTo(o.HaveOccurred())
+	}()
+	_, err = testlib.DebugNodeWithChroot(oc, leaderKcm, "mv", "/etc/kubernetes/manifests/kube-apiserver-pod.yaml", "/home/")
+	o.Expect(err).NotTo(o.HaveOccurred())
+	err = oc.AsAdmin().WithoutNamespace().Run("delete").Args("-n", "openshift-kube-apiserver", "pod/"+"kube-apiserver-"+leaderKcm).Execute()
+	o.Expect(err).NotTo(o.HaveOccurred())
+
+	err = wait.Poll(5*time.Second, 30*time.Second, func() (bool, error) {
+		output, err := oc.AsAdmin().Run("get").Args("co", "kube-controller-manager").Output()
+		if err != nil {
+			return false, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 123) but it returns nil

(nilerr)